File: Ruby/1043.rb
a, b, c = gets.chomp.split().map{ |x| x.to_f}
if (a + b > c and a + c > b and b + c > a) then
  puts ("Perimetro = %0.1f" % (a+b+c))
else
  puts ("Area = %0.1f" % ( ( (a+b) * c ) / 2 ) )
end

File: Lua/1043.lua
a = io.read("*n")
b = io.read("*n")
c = io.read("*n")
if (a + b > c and a + c > b and b + c > a) then
	print(string.format("Perimetro = %.1f", (a+b+c)))
else
	print(string.format("Area = %.1f", ( ( (a+b) * c) / 2 ) ))
end
